classes/core-image-pelux-qtauto.bbclass

```diff
@@ -0,0 +1,21 @@
+#
+#   Copyright (C) 2017 Pelagicore AB
+#   SPDX-License-Identifier: MIT
+#
+
+DESCRIPTION = "Class for inheriting a Pelux image with Qt frontend"
+
+inherit core-image-pelux
+
+# A bunch of QtAuto components
+QTAUTO_COMPONENTS := " \
+    qtapplicationmanager \
+    qtivi \
+    qtvirtualkeyboard \
+    dbus-session \
+"
+
+# Classes are special so we only do this if b2qt (which contains the components) is enabled
```

Comment:
Maybe be more explicit and say: the b2qt meta layer is used/enabled, b2qt might not be obvious what it is to others.